react-front-end/tsrc/fileuploader/InlineFileUploader.tsx

```diff
@@ -296,14 +296,15 @@ export const InlineFileUploader = ({
    */
   const buildActions = (file: UploadedFile | UploadingFile): UploadAction[] => {
     if (isUploadedFile(file)) {
+      if (!editable) return [];
```

Comment:
Good practice in C based syntax languages (so JS, TS, Java) to use curly braces even for simple statements like this to protect against accidental future editing causing modified logic errors.

```suggestion
      if (!editable) {
        return [];
      }
```